src/entity/CommonGame.java

```diff
@@ -0,0 +1,106 @@
+package entity;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class CommonGame implements Game {
+    private int ID;
+    private int score;
+    public String difficulty;
+    public String genre;
+    public int InitialLives;
+    public int max_rounds;
+    public int current_lives;
+    public int rounds_played = 0;
+    private final LocalDateTime createdAt;
+    private LocalDateTime FinishedAt;
+
+    // TODO add arguments to constructor?
+    CommonGame(String genre, String difficulty, int max_rounds,
+               int initialLives,  LocalDateTime creationTime) {
+        this.genre = genre;
+        this.difficulty = difficulty;
+        this.max_rounds = max_rounds;
+        this.InitialLives = initialLives;
+        this.createdAt = creationTime;
+    }
```

Comment:
To simplify things, we can delete the `creationTime` parameter and set `this.createdAt` to be the current time (i.e. `this.createdAt = LocalDateTime.now()`).

Also, the constructor is a good place to set a unique ID for `ID`. I know I wrote up the original interface with `ID` being an `int` but it might be easier to implement it as a random UUID String instead. Then you could simply add `this.ID = UUID.randomUUID().toString();` and be done with it. See this [link about UUID](https://docs.oracle.com/javase/8/docs/api/java/util/UUID.html) for more info.